compiler/rustc_middle/src/ty/fold.rs

```diff
@@ -721,7 +715,6 @@ impl<'tcx> TyCtxt<'tcx> {
 
 enum Direction {
```

Comment:
I assume this can be removed - but I'm not sure why you've pinged me, I don't recall having added this code (or having any experience with this area), and `git blame` appears to agree with my not-always-reliable memory in this case.

Reply:
It was lifetime-related, so I thought you'd be likely to know 😅 It's my fault for being too lazy to `git blame`.

cc @scalexm

Reply:
It can go yes

Reply:
Ok will remove it as well